app/views/index/about.phtml

```diff
@@ -45,3 +45,11 @@
 	<h2><?= _t('index.about.credits') ?></h2>
 	<p><?= _t('index.about.credits_content') ?></p>
 </main>
+
+<?php if (!FreshRSS_Auth::hasAccess()) { ?>
```

Comment:
Ah, it is maybe more this special case that I was wondering about. Isn't it fine to always show the TOS there?

Reply:
I would like to show the footer only in the case if the user is not logged in, because in this case there is no left side menu.

If the user is logged in, then the TOS is shown in the menu one line bellow. A footer does not fit into the design yet.
![grafik](https://user-images.githubusercontent.com/1645099/227774461-71d686ec-3b08-4176-902e-bd6b71e85df8.png)